### PR TITLE
feat: import and export test params

### PR DESCRIPTION
**Pull request by tsaleksandrova**

### CODE REVIEW

File: src/reporter/paramsReporter.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+function JasmineParamsReporter(config, instanceConfig, logger) {
+  this.config = config || {};
+  this.logger = logger;
+}
+
+JasmineParamsReporter.prototype.jasmineDone = function() {
+  if (this.config.exportParams) {
+    this.logger.debug('Exporting test params to file ' + this.config.exportParams);
+    fs.writeFileSync(this.config.exportParams, JSON.stringify(browser.testrunner.config.params, null, 2));
```
Comment (maximnaidenov) [inline src/reporter/paramsReporter.js:16]: as discussed, lets use separte object: testrunner.config.exportParams

File: src/reporter/paramsReporter.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+var fs = require('fs');
```
Comment (maximnaidenov) [inline src/reporter/paramsReporter.js:1]: no need to have this code as reporter, lets integrate it in the uiveri5 core

File: src/cli.js [hunk 3]
```diff
@@ -30,6 +30,8 @@ var argv = require('yargs').
   string('browsers').
   describe('browsers', 'Comma separated list of browsers to execute tests, overwrites *.conf.js values, defaults to chrome').
   describe('params', 'Param object to be passed to the tests').
+  describe('importParams', 'JSON file with input params to be passed to the tests').
```
Comment (maximnaidenov) [inline src/cli.js:33]: importParamsFile

File: src/cli.js [hunk 4]
```diff
@@ -30,6 +30,8 @@ var argv = require('yargs').
...
+  describe('exportParams', 'JSON file where output params will be written').
```
Comment (maximnaidenov) [inline src/cli.js:34]: exportParamsFile

**Comment by maximnaidenov** [comment]
Please add a scenario test

**Review by ghost — APPROVED** [review]

**Pull request merged**